README.md

```diff
@@ -9,14 +9,15 @@ Python CLI app to detect if a screenshot of a tweet is genuine. Work in Progress
 ### Setting Up
 
 * Run `python setup.py install` to install requirements. Install script is still a work in progress.
-* Run `python fakemenot/app.py` after installation.
-* Go to https://apps.twitter.com/app/new and put your consumer and API keys in `fakemenot/twitter.config`
+* Go to https://apps.twitter.com/app/new and put your consumer and API keys in
```

Comment:
Hey, thanks for the PR :)

I should have edited the wiki after the setup.py change. The current setup.py installs it to a proper path now, so instead of running  `__init.py__`, fakemenot can be directly run on the command line.

+1 about the order change. Can you please change that line?